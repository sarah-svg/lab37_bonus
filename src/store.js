import { createStore } from 'redux';
import { composeWithDevTools } from 'rex-devtools-extension';
import reducer from './reducer/gitUserReducer';


const store = createStore(reducer, composeWithDevTools);

export default store;
