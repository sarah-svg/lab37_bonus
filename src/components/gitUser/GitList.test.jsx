import React from 'react';
import { Provider } from 'react-redux';
import { render, cleanup } from '@testing-library/react';
import GitList from './GitList';
import store from '../../store';




describe('git list component', () => {
  beforeEach(() => cleanup());
  it('test the git list page', () => {
    const { asFragment } = render( 
      <Provider store={store}> <GitList/> </Provider >
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
