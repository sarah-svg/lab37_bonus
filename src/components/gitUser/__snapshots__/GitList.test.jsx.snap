// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`git list component test the git list page 1`] = `
<DocumentFragment>
   
  <ul
    data-testid="repo-list"
  />
   
</DocumentFragment>
`;
